addons/sourcemod/scripting/include/nd_commander_build.inc

```diff
@@ -0,0 +1,52 @@
+#if defined _nd_commmander_build_included_
+  #endinput
+#endif
+
+#define _nd_commmander_build_included_
+
+enum eNDStructures
+{
+    eND_CommandBunker = 0,
+    eND_MachineGunTurret,
+    eND_TransportGate,
+    eND_PowerStation,
+    eND_WirelessRepeater,
+    eND_RelayTower,
+    eND_SupplyStation,
+    eND_Assembler,
+    eND_Armory,
+    eND_Artillery,
+    eND_RadarStation,
+    eND_FlamethrowerTurret,
+    eND_SonicTurret,
+    eND_RocketTurret,
+    eND_Wall,
+    eND_Barrier
+}
```

Comment:
This is code duplication. Already present in [nd_structures.inc](https://github.com/stickz/Redstone/blob/master/addons/sourcemod/scripting/include/nd_structures.inc). We should use that instead.

Reply:
Addressed in https://github.com/stickz/Redstone/commit/082c09622def6ee0f5bf5f67b386b65eb1e9ea82